core/src/main/java/org/apache/servicecomb/core/definition/loader/SchemaListenerManager.java

```diff
@@ -74,12 +71,22 @@ public void notifySchemaListener(List<SchemaMeta> schemaMetaList) {
   }
 
   public SchemaMeta ensureFindSchemaMeta(String microserviceName, String schemaId) {
-    MicroserviceMeta microserviceMeta = microserviceMetaManager.ensureFindValue(microserviceName);
+    if (!RegistryUtils.getMicroservice().getServiceName().equals(microserviceName)) {
+      LOGGER.error("miroserviceName : {} is different from the default microserviceName :{}",
```

Comment:
This is an internally used class, will this logic happen? Or can we delete the parameter microserviceName?

Reply:
this logic will not happen in the normal mode , and I have asked jimin about whether I could delete parameter microserviceName because it's no use now. He suggested me to keep it for the compatibility of the method with 
other classes or modules and he advised me to compare the parameter microserviceName with the default right micorserviceName. just make sure the program is running as we expected。